Source/ActivityPub.Types/AS/Collection/ASOrderedCollection.cs

```diff
@@ -19,7 +19,7 @@ namespace ActivityPub.Types.AS.Collection;
 /// </remarks>
 /// <seealso href="https://www.w3.org/TR/activitystreams-vocabulary/#dfn-collection" />
 /// <seealso href="https://www.w3.org/TR/activitystreams-vocabulary/#dfn-orderedcollection" />
-public class ASOrderedCollection : ASObject, IASModel<ASOrderedCollection, ASOrderedCollectionEntity, ASObject>, IEnumerable<Linkable<ASObject>>
+public class ASOrderedCollection : ASCollection, IASModel<ASOrderedCollection, ASOrderedCollectionEntity, ASCollection>, IEnumerable<Linkable<ASObject>>
```

Comment:
this, unfortunately, won't work because ASCollection has a separate entity with a separate copy of the collection. It may be possible to support this by loading the same entity into both classes - I'll take a look into that.

Reply:
It's not that big of a deal, honestly. I'll split out the shared inbox. The rest is workable as-is. The AS2 inheritance tree is nonsensical.